nautobot/extras/forms.py

```diff
@@ -51,7 +50,14 @@
     Webhook,
 )
 from .utils import FeatureQuery
-from nautobot.extras import choices
+from nautobot.extras.choices import (
```

Comment:
For consistency with the above imports, this should probably be `from .choices import` as well.